test/integration/node-specific/comment_with_falsy_values.test.ts

```diff
@@ -1,8 +1,8 @@
 import { Long } from '../../../src';
 import { TestBuilder, UnifiedTestSuiteBuilder } from '../../tools/utils';
 
-const falsyValues = [0, false, '', Long.ZERO, null, NaN] as const;
-const falsyToString = (value: typeof falsyValues[number]) => {
+export const falsyValues = [0, false, '', Long.ZERO, null, NaN] as const;
```

Comment:
instead of exporting from a test file, should we just add this to our test utils? it's not a good idea to entangle the test files like this, but also, see my comment on the other file regarding whether we definitely need that to be seprate

Reply:
Sure, I'll move this to the utils if we decide to leave the standalone distinct test file (see my response to your comment)